build.sh

```diff
@@ -820,6 +824,8 @@ while getopts ":hacCfgijmp:q:uvslwedk:bx:S:X:" opt; do
             ;;
         d)
             PRINT_MATDBG_HELP=true
+            # TODO: Uncomment below when fgviewer is ready
+            # FGVIEWER_OPTION="-DFILAMENT_ENABLE_FGVIEWER=ON"
```

Comment:
so the matdbg option will be used to enable fgviewer? What are the thoughts behind that? (I think it's perfectly ok, but just wondering; and also wondering why not just co-opt the matdbg server for fgviewer messages).

Reply:
I think we'd want matdbg and fgdbg to be separate; like we can enable one without the other.

Reply:
Oh, I thought `-d` is for debug rather than matdbg only.
I separate the option now.